Installers/Linux/RUN/postinstall.sh

```diff
@@ -16,8 +16,18 @@ fi
 DIR=$(pwd)
 IDIR="/usr/lib/mono/xbuild/MonoGame/v3.0"
 
-# Automatic dependency installer
-./Dependencies/dependencies.sh
+# Show dependency list
+echo "Please make sure the following packages are installed:"
+echo " - monodevelop"
+echo " - libopenal-dev"
+echo " - referenceassemblies-pcl / mono-pcl"
```

Comment:
@DDReaper we are saying we need these :) Means the .Core project can be done for Linux to IMHO ;)